tests/isBotInstalled.js

```diff
@@ -1,4 +1,4 @@
-/* eslint-disable no-unused-expressions */
+/* eslint-disable no-unused-expressions,max-len */
```

Comment:
I think lint could be disabled inside tests

Reply:
changed to disable inside the tests.